test/collections.statistics.js

```diff
@@ -0,0 +1,87 @@
+(function() {
+    var _ = typeof require == 'function' ? require('..') : window._;
+    QUnit.module('Statistics');
+    QUnit.test('mean', function(assert) {
+      assert.strictEqual(_.mean(null), 0, 'can handle null/undefined');
+      assert.strictEqual(_.mean(void 0), 0, 'can handle undefined');
```

Comment:
Indentation glitch at the start

```suggestion
(function() {
  var _ = typeof require == 'function' ? require('..') : window._;
  QUnit.module('Statistics');
  QUnit.test('mean', function(assert) {
    assert.strictEqual(_.mean(null), 0, 'can handle null/undefined');
    assert.strictEqual(_.mean(void 0), 0, 'can handle undefined');
```